rockcraft/pebble.py

```diff
@@ -166,18 +166,28 @@ class Pebble:
 
     PEBBLE_PATH = "var/lib/pebble/default"
     PEBBLE_LAYERS_PATH = f"{PEBBLE_PATH}/layers"
-    PEBBLE_BINARY_PATH = "bin/pebble"
-    PEBBLE_PART_SPEC = {
+    PEBBLE_BINARY_DIR = ".rock/bin"
```

Comment:
isn't `.rock` globally defined? it should be cause we're putting the control data in there. Hardcoding it here will make it harder in the future if we decide to rename it

Reply:
That's a good idea - added a new `constants` module to hold this and future constants